libs/application/template-api-modules/src/lib/modules/templates/parental-leave/emailGenerators/assignEmployerEmail.ts

```diff
@@ -50,9 +30,47 @@ export const generateAssignEmployerApplicationEmail: AssignmentEmailTemplateGene
       },
     ],
     subject,
-    html: `<p>${body
-      .split('')
-      .map((c) => (c === '\n' ? `<br />\n` : c))
-      .join('')}</p>`,
+    template: {
+      title: subject,
+      body: [
+        {
+          component: 'Image',
+          context: {
+            src: pathToAsset('logo.jpg'),
+            alt: 'Vinnumálastofnun merki',
+          },
+        },
+        {
+          component: 'Image',
+          context: {
+            src: pathToAsset('notification.jpg'),
+            alt: 'Barn myndskreyting',
+          },
+        },
+        { component: 'Heading', context: { copy: subject } },
+        { component: 'Copy', context: { copy: 'Góðan dag.' } },
+        {
+          component: 'Copy',
+          context: {
+            copy: `Umsækjandi með kennitölu ${application.applicant} hefur skráð þig sem atvinnuveitanda í umsókn sinni.`,
+          },
+        },
+        {
+          component: 'Copy',
+          context: {
+            copy: `Ef þú áttir von á þessum tölvupósti þá getur þú smellt neðan.`,
```

Comment:
I'm not sure about the context here but this doesn't seem to be entirely correct. This would translate to 

`If you were expecting this e-mail you can click below`

Again, not sure if this matters.

Reply:
Maybe this should be "Ef þú áttir von á þessum tölvupósti þá getur þú smellt á takkann hér fyrir neðan." 🤷‍♂️

Reply:
Thanks! That's the only part I did some Google translate and forgot to proof-read it 👍